packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -138,7 +138,7 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
                 focusedArcId={this.state.focusedArcId || ''}
                 href={this.props.href!}
                 calloutId={this._calloutId}
-                valueInsideDonut={this._toLocaleString(valueInsideDonut)}
+                valueInsideDonut={this._toLocaleString(valueInsideDonut?.toString())}
```

Comment:
>toString [](http://example.com/codeflow?start=73&length=8)

why is this change needed. The tostring conversion is supposed to be performed by the toLocaleString function only.

Reply:
Reverted this change. Introduced fix in utilities.ts.